src/asciidoctor-core-api.js

```diff
@@ -67,10 +78,6 @@ Opal.Asciidoctor.AbstractBlock.$$proto.convert = function () {
   return this.$convert();
 };
 
-Opal.Asciidoctor.AbstractBlock.$$proto.delegate = function () {
```

Comment:
I honestly have no idea what this method refers to. "delegate" isn't anywhere in Asciidoctor core.